include/openenclave/internal/crypto/sha.h

```diff
@@ -66,6 +66,40 @@ oe_result_t oe_sha256_update(
  */
 oe_result_t oe_sha256_final(oe_sha256_context_t* context, OE_SHA256* sha256);
 
+/**
+ * Saves the internal state of a SHA-256 context
+ *
+ * This function saves the internal state of a SHA-256 context to H and N
+ * buffers.
+ *
+ * @param context handle of context to finalized
+ * @param H buffer to write to
+ * @param N buffer to write to
+ *
+ * @return OE_OK upon success
+ */
+oe_result_t oe_sha256_save(
+    const oe_sha256_context_t* context,
+    uint32_t* H,
```

Comment:
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says to use lower case

Reply:
Done.